package.json

```diff
@@ -122,6 +122,7 @@
     "eslint-plugin-eslint-comments": "^3.1.2",
     "eslint-plugin-eslint-plugin": "^5.0.6",
     "eslint-plugin-import": "^2.25.1",
+    "eslint-plugin-jest": "link:./lib",
```

Comment:
this was so i can test the changes on the editor directly. super useful, I can make it a separate PR if you prefer.